fragmentino/molecular_fragmenter.py

```diff
@@ -157,6 +191,19 @@ def group_fragments_by_size(self):
                 if j > i and fragment_i.same_size(fragment_j):
                     self.swap_fragments(i + 1, j)
 
+    def order_fragments_by_centrality(self):
+        """Order fragments according to centrality, i.e.,
+        with respect to increasing distance to the average of the center of mass of the fragments.
+        """
+        CM = []
+        for fragment in self:
+            CM.append(fragment.center_of_mass)
+
+        CM = np.array(CM)
+
+        order = np.argsort(np.linalg.norm(CM - np.mean(CM, axis=0), axis=1))
+        self.g.vertices = [self.g.vertices[i] for i in order]
```

Comment:
~I believe you can just do~If `vertices` was an array and not a list, you could do

```python
self.g.vertices = self.g.vertices[order]
```

But I guess vertices is a list here, so nevermind.